examples/resources/awscc_iam_oidc_provider/iam_oidc_provider.tf

```diff
@@ -0,0 +1,17 @@
+resource "awscc_iam_oidc_provider" "this" {
+  thumbprint_list = ["cf23df2207d99a74fbe169e3eba035e633b65d94"]
+  client_id_list  = ["266362248691-342342xasdasdasda-apps.googleusercontent.com", ]
+  url             = "https://accounts.google.com"
+  tags = [{
+    key   = "NAME"
+    value = "IAM OIDC Provider"
+    },
+    {
+      key   = "Environment"
+      value = "development"
+    },
+    { key   = "Modified By"
+      value = "AWSCC"
+  }]
+
+}
```

Comment:
I suggest to use example with Terraform Cloud as the OIDC provider , basically adopting this example: https://github.com/hashicorp/terraform-dynamic-credentials-setup-examples/blob/main/aws/aws.tf

```suggestion
# Data source used to grab the TLS certificate for Terraform Cloud.
data "tls_certificate" "tfc_certificate" {
  url = "https://app.terraform.io"
}

resource "awscc_iam_oidc_provider" "this" {
  thumbprint_list = [data.tls_certificate.tfc_certificate.certificates[0].sha1_fingerprint]
  client_id_list  = ["aws.workload.identity"]
  url             = data.tls_certificate.tfc_certificate.url
  tags = [
    {
      key   = "Name"
      value = "IAM OIDC Provider"
    },
    {
      key   = "Environment"
      value = "Development"
    },
    { key   = "Modified By"
      value = "AWSCC"
    }
  ]
}
```